src/ui/components/select-dropdown/test/index.js

```diff
@@ -39,15 +39,16 @@ describe( 'index', () => {
 			expect( dropdown.find( '.select-dropdown.is-open' ).length ).toBe( 0 );
 		} );
 
-		test( 'should execute toggleDropdown when clicked', () => {
-			const originalToggleDropdown = SelectDropdown.prototype.toggleDropdown;
-			SelectDropdown.prototype.toggleDropdown = jest.fn();
+		test( 'should execute onContainerClick when clicked', () => {
+			const originalOnContainerClick = SelectDropdown.prototype.onContainerClick;
+			SelectDropdown.prototype.onContainerClick = jest.fn();
 
+			const fakeEvent = prepareFakeClickEvent( { } );
```

Comment:
Can we inline the `prepareFakeClickEvent` function? I don't see it used anywhere else.